sharing.go

```diff
@@ -91,6 +91,22 @@ func (c *Sharing) ListSharedLinks(in *ListShareLinksInput) (out *ListShareLinksO
 	return
 }
 
+type RevokeSharedLinkInput struct {
+	Url            string `json:"url"`
+}
+
+
+func (c *Sharing) RevokeSharedLink(in *RevokeSharedLinkInput) (err error) {
+	
+	endpoint := "/sharing/revoke_shared_link"
+	_, err := c.call(endpoint, in)
+	if err != nil {
+		return
+	}
```

Comment:
not a big deal but could just do `return` here instead of the conditional. Mind adding a comment to the struct/method as well?

LGTM otherswise thanks!!